docs/API/content-type/creating-co.md

```diff
@@ -3,106 +3,392 @@ description: How to add Content Objects in Flotiq
 
 # Content Objects
 
-Once a <abbr title="Content Type - a model of data that has been defined inside the Content Repository.">Content Type</abbr> has been defined in the system - the user can create <abbr title="Content Object - an instance of a Content Type.">Content Objects</abbr>  of that Content Type. This is done either directly through the API or via the convenient Content Entry tools provided within the Content Management Platform.
+Once a <abbr title="Content Type - a model of data that has been defined inside the Content Repository.">Content Type</abbr> 
+has been defined in the system - the user can create 
+<abbr title="Content Object - an instance of a Content Type.">Content Objects</abbr>  of that Content Type. 
+This is done either directly through the API or via the convenient Content Entry tools provided within the 
+Content Management Platform.
 
-## Authoring content through the API
+!!! note
+    You will need to use your `Application Read and write API KEY` to perform this action
```

Comment:
I don't know what does it mean by "this action" in this context.

Reply:
Ok, it's about creating content objects, I think the title of the page "Content Objects" could be "Creating new Content Objects".